website/content/docs/commands/volume/claim-list.mdx

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+page_title: 'Commands: volume claim list'
+description: |
+  List task group host volume claims.
+---
+
+# Command: volume claim list
+
+The `volume claim list` command displays all task group host volume claims. These claims
+are used by [stateful deployments].
+
+## Usage
+
+```plaintext
+nomad volume claim list [options]
+```
+
+This command accepts an optional claim ID or prefix as the sole argument. If
+there is an exact match based on the provided volume ID or prefix, then
+information about the specific claim is queried and displayed. Otherwise, a
+list of matching claims and information will be displayed.
```

Comment:
```suggestion
This command accepts an optional claim ID or prefix as the sole argument. If
there is an exact match based on the provided volume ID or prefix, then
the command displays information about the specific claim. Otherwise, the command 
displays a list of matching claims.
```

Style: active voice, present tense